src/main/kotlin/com/jetbrains/snakecharm/lang/parser/SnakemakeLexer.kt

```diff
@@ -128,7 +128,7 @@ class SnakemakeLexer : PythonIndentingLexer() {
         }
 
         if (tokenType === PyTokenTypes.LINE_BREAK) {
-            val text = tokenText
+            val text = tokenText.substringAfterLast(System.lineSeparator())
```

Comment:
Better to check this on windows machine. And create some file on Windows and open on Linux.

I assume, that you could open in IDE launched on Unix machine some file created on windows with `\r` and vice versa. An IDE is likely to render the file correctly although separators are wrong.